runtime/context.go

```diff
@@ -245,18 +260,18 @@ func (c *ContextExtractors) ExtractLogFields(ctx context.Context) []zap.Field {
 
 // ContextLogger is a logger that extracts some log fields from the context before passing through to underlying zap logger.
 type ContextLogger interface {
-	Debug(ctx context.Context, msg string, fields ...zap.Field)
-	Error(ctx context.Context, msg string, fields ...zap.Field)
-	Info(ctx context.Context, msg string, fields ...zap.Field)
-	Panic(ctx context.Context, msg string, fields ...zap.Field)
-	Warn(ctx context.Context, msg string, fields ...zap.Field)
+	Debug(ctx context.Context, msg string, fields ...zap.Field) context.Context
```

Comment:
Now that we are returning a `context`, Please update the comment on line 261 about the new responsibility for the interface and what implementers should take care of while returning a `context`